doc/api/dgram.md

```diff
@@ -226,6 +241,24 @@ never have reason to call this.
 If `multicastInterface` is not specified, the operating system will attempt to
 drop membership on all valid interfaces.
 
+### socket.dropSourceSpecificMembership(sourceAddress, groupAddress[, multicastInterface])
+<!-- YAML
+added: REPLACEME
+-->
+
+* `sourceAddress` {string}
+* `groupAddress` {string}
+* `multicastInterface` {string}
```

Comment:
Maybe the documentation should mention that this is optional (for both methods)?

Reply:
I'm not terribly familiar with Node documentation. From what I've observed, the function signature indicates the optionality arguments by using square brackets e.g. `[arg]`. Note that the source-specific API is very close to the existing non-source-specific one. In both, the function description implies the optionality of `multicastInterface` rather than being fully explicit.

What are you suggesting for me to do?